en/04_Changelogs/5.2.0.md

```diff
@@ -11,6 +11,7 @@ title: 5.2.0 (unreleased)
   - [ErrorPage allowed codes configuration](#errorpage-allowed-codes-configuration)
   - [Create random passwords for new users](#create-random-passwords-for-new-users)
   - [Buttons to select all files and deselect all files](#bulk-action-buttons)
+  - [SearchableDropdownField components](#searchable-dropdown-field)
```

Comment:
```suggestion
  - [New searchable dropdown fields](#searchable-dropdown-field)
```
"Component" doesn't really make sense here, since that makes it sound like it's just a react thing. And mentioning the class isn't useful for people who have never seen the class before (which will be anyone reading the changelog :p)

Reply:
Updated